README.md

```diff
@@ -4,32 +4,30 @@
 
 [Website](http://ss13.moe) - [Code](https://github.com/vgstation-coders/vgstation13)
 
-[IRC](irc://irc.rizon.net/vgstation) (irc.rizon.net #vgstation), here is an embedded link to it:  [![Visit our IRC channel](https://kiwiirc.com/buttons/irc.rizon.net/vgstation.png)](https://kiwiirc.com/client/irc.rizon.net/?nick=Newcomer|?&theme=basic#vgstation)
-
 Discord Coding Server Invite Link:
 No longer available from github due to spambots using it.
-Ask on the /vg/ thread for a link or join the IRC at the link above and ask for an invite, there's a bot that relays messages between IRC and discord.
+Check the thread on /vg/ for a link or ask a player in-game for an invite.
 
 ---
 
 ### GETTING THE CODE
-The simplest but least useful way to obtain the code is using the Github .zip feature. You can click [here](https://github.com/vgstation-coders/vgstation13/archive/Bleeding-Edge.zip) to get the latest stable code as a .zip file, then unzip it to wherever you want. Alternatively, a much more useful method is to use a git client, the process for getting and using one is described below, (for more information our coders in IRC can tell you how to use one).
+The simplest but least useful way to obtain the code is using the Github .zip feature. You can click [here](https://github.com/vgstation-coders/vgstation13/archive/Bleeding-Edge.zip) to get the latest stable code as a .zip file, then unzip it to wherever you want. This is mostly useful for people looking to get assets from the server or to host a quick short-term game between friends.
 
-### Git client
+Alternatively, the code can be acquired via the use of a Git client as described below. If you're still having difficulty, reach out to one of our coders on Discord and they should be happy to help.
```

Comment:
> reach out to one of our coders on Discord 
fuck off

Reply:
okay where do you intend for them to reach out to them instead

Reply:
Personally, I'd have left out the direct approach and went with something like "ask on discord", but I don't mind either

and pedr is probably just saying FUCK OFF LEAVE ME ALONE

Reply:
the thread?
a github discussion?

i'll admit my hatred for "login to see this" may be clouding my judgement but still:
discordites FUCK OFF